server/src/main/java/com/skillmagnet/Enrolls/EnrollsController.java

```diff
@@ -0,0 +1,91 @@
+package com.skillmagnet.Enrolls;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.skillmagnet.Course.Course;
+import com.skillmagnet.Course.CourseRepository;
+import com.skillmagnet.User.User;
+import com.skillmagnet.User.UserRepository;
+
+@RestController
+@RequestMapping("api")
+public class EnrollsController {
+    /*
+     * Using all repository's to find all objects involved in relation
+     */
+
+    @Autowired
+    CourseRepository courseRepository;
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Autowired
+    EnrollsRepository enrollsRepository;
+
+    /**
+     * Find's all enrollments based on user id
+     * 
+     * Each item in the return list displays the full course object
+     * @param id - user id
+     * @return a List of enrollments
+     */
+    @GetMapping("/enrolls/{uid}")
+    public ResponseEntity<List<Enrolls>> getUserEnrolledCourses(@PathVariable("uid") int id){
+        User requestedUser = userRepository.findById(id);
+        if (requestedUser == null){
+            return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
+        }
+        return new ResponseEntity<>(enrollsRepository.findByEnrolledUser(requestedUser), HttpStatus.OK);
+    }
+
+    /**
+     * Creates a new enrollment relation
+     * 
+     * Finds user and course from path variables & creates Enrollment obj
+     * setting the relations to the found objects
+     * @param uid - user ID
+     * @param cid - course ID
+     * @return - new Enrollment that's been saved
+     */
+    @PostMapping("/enrolls/{uid}/{cid}")
+    public ResponseEntity<Enrolls> createEnrollment(@PathVariable("uid") int uid, @PathVariable("cid") int cid){
```

Comment:
`POST` should not have parameters in url

Make it take in a `@RequestBody` and either use an `Enrolls` object to grab the user + course IDs or create a special request object like [this](https://github.com/nickerick/skill-magnet/blob/73ff6278bd90fc8a383f960e7ab80646c79f6b6f/server/src/main/java/com/skillmagnet/Lesson/LessonController.java#L61)